prisma/schema.prisma

```diff
@@ -7,24 +7,25 @@ generator client {
 
 datasource db {
     provider  = "postgresql"
-    // NOTE: When using mysql or sqlserver, uncomment the @db.Text annotations in model Account below
-    // Further reading:
-    // https://next-auth.js.org/adapters/prisma#create-the-prisma-schema
-    // https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#string
     url       = env("DATABASE_URL")
     directUrl = env("DIRECT_URL")
 }
 
-model Post {
-    id        Int      @id @default(autoincrement())
-    name      String
-    createdAt DateTime @default(now())
-    updatedAt DateTime @updatedAt
-
-    createdBy   User   @relation(fields: [createdById], references: [id])
-    createdById String
-
-    @@index([name])
+// User model for authentication and user-specific data
```

Comment:
These comments are self-explanatory. Comments in a prisma file are generally unneeded, as the developer should understand the shape of SQL data. 
example: Given a new or deprecated field, a NOTE: this field is deprecated